### ISSUE TITLE
Disable timeline real-time updates

### ISSUE THREAD

**Issue by hoodiek** [tootsuite/mastodon#240]
since we're under heavy load, and things aren't working very well, it's pointed out a problem.

i'd suggest a manual "update feed" button to resolve the issue with this, as well as options to maybe turn off automatic update for those who would rather do things piece at a time. 

also a "refresh every (textbox or selector goes here for how many) seconds" would be appreciated.

thank you for listening.

**Comment by Gargron** [comment]
Adding settings for specifically the web UI will be a complicated task, I'm not keen on mixing them in with account or profile settings since those are independent of the UI you're using. But the thing is once web UI-specific settings will become a thing *somehow*, it'll pave way for re-ordering columns, adding new columns, choosing which types of notifications to show etc etc.

**Comment by hoodiek** [comment]
sounds like a big task, but you seem up to it. take care, friend. thank you for listening.

**Comment by himselfv** [comment]
It's hard to click at anything precisely when toots are being shifted constantly by the stream of updates. The way twitter does this is much better, it shows the number of new tweets at the top and a button to download them.

It's also excessive to update this often. Even if there's a toot every second, people can't read at this speed. (At least most of us). One update every 30 seconds should be enough, unless the user specifically wants faster updates.

**Comment by himselfv** [comment]
Is this fixed?

**Comment by Gargron** [comment]
I have closed all issues opened by this user due to this comment: https://pleroma.site/objects/a00611de-f0ac-454e-ade0-cf949195f0fa

Normally, when people enter into a financial transaction, they do so knowingly: "I will do x in exchange for y", and the other person may agree or disagree. The user in question has submitted a whole array of feature requests, and way after that decided that they need reimbursement for it, even though there was no such agreement in advance. I see no other way than to close all feature requests by that user, and not use them.

On top of that, the user in question has claimed to be the inventor of features that multiple other people wanted, that there existed prior art for (e.g. Archive of Our Own, e-mail subject lines, whole community has been using ROT13 with a line of cleartext for the subject), and that other people have actually put in the work to implementing. For example, content warnings have been implemented in #460, and changed, polished and iterated on multiple times since, if you follow the code history.

In my view, the nature of bug reports and feature requests is mutually beneficial to the reporter and the project maintainer: You say what you want, and in the best case you get what you wanted. The project becomes better and your personal need is addressed. But in case you require more compensation than that, you must state so beforehand to give the project maintainer a chance to consent (and be prepared to hear a "no").

**Comment by hoodiek** [comment]
i did not ask you to close my issues. i stated that i have not recieved credit for what i contributed, because i haven't. you hoard all credit for everything people contribute to you. i did not state that i demanded payment for my efforts, simply that i did not receive them, because i haven't

for you to close all my issues is hostile to your userbase, and i did not request it or require it. there is no reason for this, as far as i can tell, other than that you feel offended that i do not enjoy your method of operation

**Comment by hoodiek** [comment]
@Gargron i would advise reopening the issues in question, as there was no reason for the closure other than 'some words i said, that you didn't like'. i will never be contributing to your project again, but your users do not need to suffer due to the bad blood between us. goodbye, and i hope to never meet you again. do not let that color your opinion of your users, or change how you treat them, however. they deserve all the best, regardless of whether or not you think i am in the wrong for having found issue with your method of operation
